init.sqf

```diff
@@ -99,6 +99,14 @@ f_script_briefing = [] spawn f_fnc_createBriefing;
 
 // ====================================================================================
 
+// F3 - Combat Life Saver EH
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// Uncomment the line below to enable the Combat Life Saver heal handler. Does nothing unless you have player units using the "cls" assignGear role.
+
+// [] execVM "f\medical\f_CLSEH.sqf";

```

Comment:
Would it make sense to leave this on by default? It doesn't really do anything bad if you don't assign CLSs, aside from add a tiiiny performance overhead.

Reply:
I'd prefer to leave it off by default since it would do literally nothing in the standard ORBAT, which is still what most of our missions use. It's not a lot of performance overhead, but it is _some_, and anyone who wants to use this is already deep enough to be making semi-complex ORBAT changes.